docs/StackedFilters.md

```diff
@@ -346,37 +351,39 @@ import {
 } from '@mui/material';
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
 
-const postListFilters: FiltersConfig = {
+const PostListFilters: FiltersConfig = {
```

Comment:
You should keep the `PostListFilters` starting with a lowercase, as it is a configuration object and not a React Component

```suggestion
const postListFilters: FiltersConfig = {
```